io.github.dynobo.sphereview.yml

```diff
@@ -0,0 +1,59 @@
+app-id: io.github.dynobo.sphereview
+command: sphereview
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.node24
+build-options:
+  env:
+    CARGO_HOME: /run/build/rust-flatpak/cargo
```

Comment:
```suggestion
```

Reply:
Done in [20b53ba](https://github.com/flathub/flathub/pull/6675/commits/20b53ba4683f4bfe922ba8847e05f1401e2b7f13)